### PR TITLE
Upgrade to Electron 2

### PR DESCRIPTION
**Pull request by manuq**
Point to hatch-previewer latest commit that was upgraded to Electron
2.0.x series. NPM sources were regenerated from the package-lock file.

Also:
- Upgrade runtime to latest freedesktop
- Upgrade NodeJs to latest stable

https://phabricator.endlessm.com/T23700

### CODE REVIEW

**Comment by manuq** [comment]
Note: this PR depends on this upstream PR: https://github.com/endlessm/hatch-previewer/pull/12

File: com.endlessm.HatchPreviewer.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
     "app-id": "com.endlessm.HatchPreviewer",
     "runtime": "org.freedesktop.Platform",
-    "runtime-version": "1.6",
-    "base": "io.atom.electron.BaseApp",
-    "base-version": "stable",
-    "branch": "stable",
+    "runtime-version": "18.08",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "18.08",
+    "branch": "wip",
```
Comment (nedrichards) [inline com.endlessm.HatchPreviewer.json]: I understand what you're doing here, but this won't get through the flathub build system ;-) switch it back and then I can review and merge after the upstream PR

Comment (manuq) [reply com.endlessm.HatchPreviewer.json]: I was testing you! (not). Good catch, thanks.

**Comment by nedrichards** [comment]
bot, build

**Pull request merged**